app/forms/hyrax/forms/administrative_set_form.rb

```diff
@@ -27,7 +27,9 @@ class AdministrativeSetForm < Valkyrie::ChangeSet
       property :date_modified, readable: false
       property :date_uploaded, readable: false
 
-      property :depositor
+      property :creator
```

Comment:
AdminSet uses `creator`.  I don't know if `depositor` is also required.  I couldn't find it used anywhere, but may have missed something.